cpp/include/legate_dataframe/core/column.hpp

```diff
@@ -603,27 +631,33 @@ class PhysicalColumn {
    * @brief Move local cudf column into this unbound physical column
    *
    * @param column The cudf column to move
+   * @param allow_copy If true, copies the data if the column is bound.
    */
-  void move_into(std::unique_ptr<cudf::column> column);
+  void move_into(std::unique_ptr<cudf::column> column, bool allow_copy = false);
```

Comment:
So move into will lazily try to move the input if it can, and otherwise copy into any existing buffer?

Is there any argument for having a move_into and copy_into function to make this explicit?

Reply:
Yeah, you are right, that is nicer API wise.  I tried to avoid branching in the tasks for now.  But I guess a:
```
if get_():
    tbl.copy_into();
else:
    tbl.move_into();
```
at the end of a few tasks isn't bad since it is likely not quite permanent anyway.